plugin/src/main/java/com/denizenscript/denizen/events/block/FurnaceBurnsItemScriptEvent.java

```diff
@@ -35,6 +34,9 @@ public class FurnaceBurnsItemScriptEvent extends BukkitScriptEvent implements Li
 
     public FurnaceBurnsItemScriptEvent() {
         registerCouldMatcher("furnace burns <item>");
+        this.<FurnaceBurnsItemScriptEvent, DurationTag>registerDetermination(null, DurationTag.class, (evt, context, time) -> {
+            evt.event.setBurnTime(time.getTicksAsInt());

```

Comment:
I believe this is a breaking change? Previously inputting just a number would go through the `ElementTag` handling and be set as ticks, now it'll be converted into a `DurationTag` and be set as seconds (unless `setBurnTime` is in seconds instead of ticks for some reason).